charts/prometheus-pushgateway/README.md

```diff
@@ -50,6 +50,24 @@ $ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### To 2.0.0
+
+__!!! BREAKING CHANGE !!!__
+
+Version 2.0.0 adapted [Helm label and annotation best practices](https://helm.sh/docs/chart_best_practices/labels/). Specifically, labels mapping is listed below:
+
+```console
+OLD                 => NEW
+----------------------------------------
+heritage            => app.kubernetes.io/managed-by
+chart               => helm.sh/chart
+[container version] => app.kubernetes.io/version
+app                 => app.kubernetes.io/name
+release             => app.kubernetes.io/instance
+```
+
+Because select labels are immutable, `helm upgrade` cannot be supported. Please reivew and save the existing labels and adapt to the new labels. Then, [uninstall](#uninstall-chart) and [reinstall](#install-chart) the new chart.
```

Comment:
It might be enouhgt to delete the statefulset before upgrading.
Like here: https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-node-exporter#3x-to-4x

Reply:
@monotek 
But by default, prometheus-pushgateway is installed with Deployment only. See:

https://github.com/prometheus-community/helm-charts/blob/01875d567cf238cebfca9d7bb49b8aa6ebb1f08f/charts/prometheus-pushgateway/values.yaml#L177